doc/source/howtos/upgrade_guide.rst

```diff
@@ -117,3 +120,41 @@ To upgrade the cluster, follow these steps for each node:
 #. Repeat this procedure for the next node in the cluster
    until you upgrade all nodes.
 
+It is important that when rejoining, the node should synchronize using
+:term:`IST`.
+
+When performing any upgrade (major or minor), :term:`SST` could
+be initiated by the joiner node after the upgrade if the server
+was offline for some time. After :term:`SST` completes, the data
+directory structure needs to be upgraded (using mysql_upgrade)
+once more time to ensure compatibility with the newer version
+of binaries.
+
+   .. note:: In case of :term:`SST` synchronization,  messages log
+             file contains statements like "Check if state gap can
+             be serviced using IST ... State gap can't be serviced
+             using IST. Switching to SST" instead of "Receiving
+             IST: ..." lines appropriate to  :term:`IST`
+             synchronization.
+
+
+To upgrade data directory structure after :term:`SST`, following
+additional steps should be done:
+
+11. shutdown node that rejoined cluster using :term:`SST`:
+
+    .. code-block:: bash
+
+       $ sudo service mysql stop
+
+#. restart the node in standalone mode by setting the
+   :variable:`wsrep_provider` variable to ``none``, e.g.:
+
+   .. code-block:: bash
+
+      sudo mysqld --skip-grant-tables --user=mysql --wsrep-provider='none'
```

Comment:
If there are customer client instances trying to access the IP of this server, will they be able to connect when the seerver is in this mode? That could be a security risk? @kbauskar thoughts?

Reply:
I would call it more like an inconsistency risk. If the server comes for few minutes and existing application connects to it and start firing workload that is risky.
(Of-course this problem is more generic and I recall a bug that was floating around it wherein a node of the cluster boots in standalone mode and application start firing workload w/o probing for cluster status).
While the generic problem is something for upstream to fix we can propose to use --skip-networking so that user/admin is forced to connect only through local socket (less likely option for remote application)

Reply:
"To prevent any users from accessing this node while performing work on it, you may add [--skip-networking:LINK IT TO MYSQL UPSTREAM MANUAL?] to the startup options and use a local socket to connect, or alternatively you may want to divert any incoming traffic from your application to other operational nodes."